src/stories/vocabulary_icons.stories.js

```diff
@@ -0,0 +1,58 @@
+export default {
+  title: 'Vocabulary Icons',
+  decorators: [
+    storyFn => `<div class="has-text-black" style="font-size: 2em;">${storyFn()}</div>`
+  ]
+}
+
+export const Chevrons = () => `
```

Comment:
Can you rename these icons to `arrows`? `chevrons`  are less obvious and have connotation with the oil company and military symbols

Reply:
but those symbols are called chevrons in most of the webfonts (even font awesome), I think we should keep it with that name

Reply:
I too think "chevron" is clearer. Arrows are more generic.

Reply:
There are a lot of [types of pointing shapes](https://fontawesome.com/icons?d=gallery&c=arrows&m=free). I simply used `chevron` as that is what FA called them.